### PR TITLE
Backup stats data

### PR DESCRIPTION
**Pull request by alexgex**
Added module that makes daily backup of SKALE stats data and transfers it to AWS S3 bucket.

### CODE REVIEW

File: src/utils/backup.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    AWS_S3_BUCKET_NAME, AWS_REGION, AWS_ACCESS_KEY, AWS_SECRET_KEY, DATA_DIR, UPTIME_KUMA_URL,
+    RETRY_DELAY, RETRY_ATTEMPTS_COUNT
+)
+from src.utils.meta import get_last_backup_date, update_last_backup_date
+from tenacity import retry, stop_after_attempt, wait_fixed
+
+logger = logging.getLogger(__name__)
+
+TAR_FILE = 'stats_backup.tar'
+FILES = ["stats-dump.db", "meta-dump.json", "network-stats.json"]
```
Comment (DmytroNazarenko) [inline src/utils/backup.py]: Use '' instead of ""

File: docker-compose.yaml [hunk 2]
```diff
@@ -13,6 +13,11 @@ services:
       PROXY_DOMAIN: ${PROXY_DOMAIN}
       ETH_API_KEY: ${ETH_API_KEY}
       SCHAIN_NAMES: ${SCHAIN_NAMES}
+      AWS_S3_BUCKET_NAME: ${AWS_S3_BUCKET_NAME}
+      AWS_REGION: ${AWS_REGION}
+      AWS_ACCESS_KEY: ${AWS_ACCESS_KEY}
+      AWS_SECRET_KEY: ${AWS_SECRET_KEY}
+      UPTIME_KUMA_URL: ${UPTIME_KUMA_URL
```
Comment (DmytroNazarenko) [inline docker-compose.yaml]: Typo

**Review by DmytroNazarenko — CHANGES_REQUESTED** [review]

File: src/utils/backup.py (hunk 1 above)
Comment (alexgex) [reply src/utils/backup.py]: done

File: docker-compose.yaml (hunk 2 above)
Comment (alexgex) [reply docker-compose.yaml]: Moved to env file

**Review by DmytroNazarenko — APPROVED** [review]

**Pull request merged**